src/main/java/uk/gov/hmcts/reform/et/syaapi/service/pdf/PdfMapperService.java

```diff
@@ -0,0 +1,340 @@
+            Optional.ofNullable(StringUtils.leftPad(String.valueOf(dob.getDayOfMonth()),
+                2, "0")));
+        printFields.put(PdfMapperConstants.Q1_DOB_MONTH,
+            Optional.ofNullable(StringUtils.leftPad(String.valueOf(dob.getMonthValue()),
+                2, "0")));
+        printFields.put(PdfMapperConstants.Q1_DOB_YEAR, Optional.of(String.valueOf(dob.getYear())));
+        printFields.put(PdfMapperConstants.Q1_SEX, Optional.ofNullable(caseData.getClaimantIndType().getClaimantSex()));
+        printFields.put(String.format(PdfMapperConstants.QX_HOUSE_NUMBER, CLAIMANT_ADDRESS_PREFIX),
+            Optional.ofNullable(caseData.getClaimantType().getClaimantAddressUK().getAddressLine1()));
+        printFields.put(String.format(PdfMapperConstants.QX_STREET, CLAIMANT_ADDRESS_PREFIX),
+            Optional.ofNullable(caseData.getClaimantType().getClaimantAddressUK().getAddressLine2()));
+        printFields.put(String.format(PdfMapperConstants.QX_POST_TOWN, CLAIMANT_ADDRESS_PREFIX),
+            Optional.ofNullable(caseData.getClaimantType().getClaimantAddressUK().getPostTown()));
+        printFields.put(String.format(PdfMapperConstants.QX_COUNTY, CLAIMANT_ADDRESS_PREFIX),
+            Optional.ofNullable(caseData.getClaimantType().getClaimantAddressUK().getCounty()));
+        printFields.put(String.format(PdfMapperConstants.QX_POSTCODE, CLAIMANT_ADDRESS_PREFIX),
+            Optional.ofNullable(caseData.getClaimantType().getClaimantAddressUK().getPostCode()));
+        printFields.put(String.format(PdfMapperConstants.QX_PHONE_NUMBER, "1.6"),
+            Optional.ofNullable(caseData.getClaimantType().getClaimantPhoneNumber()));
+        printFields.put(PdfMapperConstants.Q1_MOBILE_NUMBER,
+            Optional.ofNullable(caseData.getClaimantType().getClaimantMobileNumber()));
+        printFields.put(PdfMapperConstants.Q1_EMAIL,
+            Optional.ofNullable(caseData.getClaimantType().getClaimantEmailAddress()));
+        String contactPreference = caseData.getClaimantType().getClaimantContactPreference();
+        if ("Email".equals(contactPreference)) {
+            printFields.put(PdfMapperConstants.Q1_CONTACT_EMAIL, Optional.of(contactPreference));
+        } else if ("Post".equals(contactPreference)) {
+            printFields.put(PdfMapperConstants.Q1_CONTACT_POST, Optional.of(contactPreference));
+        }
+        return printFields;
+    }
+
+    private Map<String, Optional<String>> printRespondantDetails(CaseData caseData) {
+        Map<String, Optional<String>> printFields = new ConcurrentHashMap<>();
+        List<RespondentSumTypeItem> respondentSumTypeList = caseData.getRespondentCollection();
+        if (respondentSumTypeList.size() >= MULTIPLE_RESPONDENTS) {
+            printFields.put(PdfMapperConstants.Q2_OTHER_RESPONDENTS, Optional.of(YES));
+        }
+        for (int i = 0; i < respondentSumTypeList.size(); i++) {
+            if (i >= MAX_RESPONDENTS) {
+                break;
+            }
```

Comment:
Would it not be nicer if we did this instead?
`for (int i = 0; i < respondentSumTypeList.size() && i < MAX_RESPONDENTS ; i++) {`